toolchain/check/testdata/interop/cpp/function/arithmetic_types_bridged.carbon

```diff
@@ -346,6 +345,52 @@ fn F() {
   //@dump-sem-ir-end
 }
 
+// ============================================================================
+// unsigned int param
+// ============================================================================
+
+// --- unsigned_int_param.h
+
+auto foo(unsigned int a) -> void;
+
+// --- import_unsigned_int_param.carbon
+
+library "[[@TEST_NAME]]";
+
+import Cpp library "unsigned_int_param.h";
+
+fn F() {
+  //@dump-sem-ir-begin
+  Cpp.foo(1);
+  //@dump-sem-ir-end
```

Comment:
The SemIR dumps below aren't incredibly useful because the SemIR for the thunk itself is omitted. I think it'd be useful to add a test file with just a single testcase in it, with full SemIR output enabled, so that we have testing of the SemIR produced for a thunk.

It'd also be good to have some test coverage for the Clang AST built for a thunk. Maybe you could add ... something ... to include a Clang AST dump in the output?

Reply:
We already have a test that shows the thunk in `full_semir.carbon` (`import_short_param.carbon`).
I can take it out to a separate file if you think it's useful.

Regarding the test coverage for the Clang AST, I completely agree.
I've added a TODO for that in the test.
Let's discuss how best to do that.